repo/packages/Z/zeppelin/6/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/zeppelin/assets/icon-service-zeppelin-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/zeppelin/assets/icon-service-zeppelin-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/zeppelin/assets/icon-service-zeppelin-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "zeppelin": "maibornwolff/zeppelin:0.7.3-2-all"
```

Comment:
@swoehrl-mw  Is this image publicly accessible? (Not sure if I am looking at the wrong page but i cant see any image with this tag at https://hub.docker.com/u/maibornwolff/)

Reply:
@takirala The image is publicly available from https://hub.docker.com/r/maibornwolff/zeppelin/

Reply:
@swoehrl-mw i could see the image but the tag is not visible (or is incorrect). I caught this because an internal CI failed when trying to build this docker image. 

These are all the tags that I can see : 

<img width="939" alt="screen shot 2018-02-06 at 11 38 31 am" src="https://user-images.githubusercontent.com/2296947/35880317-75f14112-0b32-11e8-84f9-c785e5515ff4.png">

docker pull also fails : 
```
> docker --debug pull maibornwolff/zeppelin:0.7.3-2-all
Error response from daemon: manifest for maibornwolff/zeppelin:0.7.3-2-all not found
```
Am I missing something ?

Reply:
@takirala If you check the marathon.json.mustache you can see that the image is defined as "{{resource.assets.container.docker.zeppelin}}-{{spark.version}}". So depending on the spark version selected in the options another image is used as I want to provide images with different spark versions (at the moment only 2.2.1-hadoop-2.6/2.7 as it is the current version).
I agree that it is a little confusing that the image name defined in the resources is not the full image/tag. Do you know of a better way to handle this?

Reply:
@ryadav88 This would fail the package.dcos build CI. cc : @jsancio Is it okay to ignore the CI build failure for this case ? we won't be able to generate a package.dcos for zepplin.

Reply:
@swoehrl-mw The problem with this solution is that we can't build `package.dcos` or Local Universe. This means that users won't be to run zeppelin in an air gapped cluster.

One possible solution is to enumerate all of the images that you need in `{{resource.assets.container.docker...}}` and picking the right one based on the version of spark. Another solution is to have a different package for each version of Spark just like we do with Cassandra, Kafka, etc.